### PR TITLE
should be fine

### PR DESCRIPTION
**Pull request by joordas**

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -21,54 +25,60 @@ def destroy
 
   def create
     @event = Event.new(event_params)
-    @event.course = params[:id]
```
Comment (Loedn) [inline app/controllers/events_controller.rb]: @joordas sure?

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -21,54 +25,60 @@ def destroy
...
-    @event.user = current_user
```
Comment (Loedn) [inline app/controllers/events_controller.rb]: +1

File: app/views/shared/_course_show_recap.html.erb [hunk 3]
```diff
@@ -1,23 +1,40 @@
...
+    <div class="recap--container__details">
+      <div class="recap-details-box">
+        <p class="recap-label">invite your friends:</p>
+        <div class="recap-invites">
+        </div>
+          <div class="event-checkbox-container">
 
-  <div class="recap--container__details">
-    <div class="recap-details-box">
-      <p class="recap-label">invited:</p>
```
Comment (Loedn) [inline app/views/shared/_course_show_recap.html.erb]: sure?

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (joordas) [reply app/controllers/events_controller.rb]: nah. I had it fixed on mine but you said @jykc did this part so I reverted. I'll put the fix back in my next PR

File: app/views/shared/_course_show_recap.html.erb (hunk 3 above)
Comment (joordas) [reply app/views/shared/_course_show_recap.html.erb]: probably. show recap is fine

**Pull request merged**